src/app/shared/issue-pr-card/issue-pr-card-milestone/issue-pr-card-milestone.component.ts

```diff
@@ -8,6 +8,7 @@ import { Milestone } from '../../../core/models/milestone.model';
 })
 export class IssuePrCardMilestoneComponent {
   @Input() milestone: Milestone;
+  @Input() hasMilestones: boolean;
```

Comment:
Can be better named as `repoHasMilestones`? Otherwise, it might be interpreted as whether this issue has milestone(s)